packages/plus/src/components/FAQ/__stories__/index.stories.tsx

```diff
@@ -0,0 +1,9 @@
+import type { Meta } from '@storybook/react'
+import { FAQ } from '..'
+
+export default {
+  component: FAQ,
+  title: 'Plus/Compositions/FAQ',
+} as Meta
```

Comment:
prefere`satisfies` instead of `as`, like that you get typescript error when you change the type :)